raspiaudio-muse-luxe.yaml

```diff
@@ -43,6 +59,39 @@ external_components:
 
 es8388:
 
+globals:
+  - id: wifi_connected
+    type: bool
+    initial_value: "false"
+    restore_value: false
+
+interval:
+  - interval: 1s
+    then:
+      - if:
+          condition:
+            and:
+              - lambda: 'return !id(wifi_connected);'
+              - wifi.connected:
+          then:
+            - globals.set:
+                id: wifi_connected
+                value: "true"
+            - light.turn_on:
+                id: top_led
+                effect: pulse
+                red: 0%
+                green: 100%
+                blue: 0%
+            - delay: 1s
+            - light.turn_off: top_led
+
+output:
+  - platform: gpio
+    id: dac_mute
+    pin: GPIO21
+    inverted: true
```

Comment:
While this may be fixing the pop, using the same GPIO pin in two different places in YAML is not a good idea and can cause conflicts.